blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitPipelineCreateRequest.java

```diff
@@ -95,4 +98,45 @@ public BluePipeline create(Reachable parent) throws IOException {
         return null;
     }
 
+    private void validate(String name, BlueScmConfig scmConfig){
+        if(scmConfig == null){
+            throw new ServiceException.BadRequestExpception(new ErrorMessage(400, "Failed to create Git pipeline")
```

Comment:
See the line below, this is generic error message. see the line below for field specific error message.